src/ConfigManager.cpp

```diff
@@ -192,26 +212,29 @@ void ConfigManager::setWifiConnectInterval(const int interval) {
 }
 
 bool ConfigManager::wifiConnected() {
+  return WiFi.status() == WL_CONNECTED;
+}
+
+bool ConfigManager::wifiConnect() {
```

Comment:
The name gives the impression it will try connect. Perhaps `isWifiConnected` is better, or `checkWifiConnection`?

Reply:
I knew you were gonna say something .. ;)

I didnt like the half connect in `begin` and half elsewhere ... How about we fold all the connection handling into `wifiConnect` --- as submitted?